test_common/harness/imageHelpers.cpp

```diff
@@ -924,6 +922,8 @@ float get_max_absolute_error(const cl_image_format *format,
 #ifdef CL_SFIXED14_APPLE
         case CL_SFIXED14_APPLE: return 0x1.0p-14f;
 #endif
+        case CL_UNORM_SHORT_555:
+        case CL_UNORM_SHORT_565: return 1.0f / 31.0f;
```

Comment:
Do we care the the error bounds should be tighter (1.0f / 63.0f?) for the channel with six bits for CL_UNORM_SHORT_565?

Reply:
I am not sure, but it would mean reworking the CTS quite a lot to have it properly implemented.

Reply:
Note that an (incorrect) attempt at the actual spec requirement for conversions is already in place in `get_max_relative_error` and includes these image types. The calling code uses this `get_max_absolute_error` function to override those limits by zeroing out any error within these (very loose) bounds.